src/Main.java

```diff
@@ -7,77 +9,143 @@
-        if( w1 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 19, 82, 2);
+        g2d.setColor(Color.WHITE);
+        g2d.fillRect(0, 0, img.getWidth(), img.getHeight());
+        g2d.dispose();
+    }
+
+    static void drawNumber(BufferedImage img, char number, int offset) {
+        if (number == '0') {
+            drawLine(img, offset, true, false, true, true, true, true, true, false, false, false);
         }
-        if( w2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 82, 2);
+        if (number == '1') {
+            drawLine(img, offset, false, false, false, false, false, true, true, false, false, false);
         }
-        if( w3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 179, 82, 2);
+        if (number == '2') {
+            drawLine(img, offset, true, true, true, false, true, true, false, false, false, false);
         }
-        if( h1 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 19, 2, 82);
+        if (number == '3') {
+            drawLine(img, offset, true, true, true, false, false, true, true, false, false, false);
         }
-        if( h2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 2, 82);
+        if (number == '4') {
+            drawLine(img, offset, false, true, false, true, false, true, true, false, false, false);
         }
-        if( h3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 19, 2, 82);
+        if (number == '5') {
+            drawLine(img, offset, true, true, true, true, false, false, true, false, false, false);
         }
-        if( h4 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 99, 2, 82);
+        if (number == '6') {
+            drawLine(img, offset, true, true, true, true, true, false, true, false, false, false);
+        }
+        if (number == '7') {
+            drawLine(img, offset, true, false, false, false, false, true, true, false, false, false);
+        }
+        if (number == '8') {
+            drawLine(img, offset, true, true, true, true, true, true, true, false, false, false);
+        }
+        if (number == '9') {
+            drawLine(img, offset, true, true, true, true, false, true, true, false, false, false);
         }
     }
 
-    public static void main(String[] args) throws IOException {
-        BufferedImage img = new BufferedImage(100, 200, BufferedImage.TYPE_INT_RGB);
+    static void drawSymbol(BufferedImage img, char number, int offset) {
+        if (number == '-') {
+            drawMinus(img, offset);
+        }
+        if (number == '+') {
+            drawPlus(img, offset);
+        }
+        if (number == '=') {
+            drawEqual(img, offset);
+        }
+    }
+
+    static void drawLine(BufferedImage img, int offset, boolean v1, boolean v2, boolean v3, boolean h1, boolean h2, boolean h3, boolean h4, boolean e1, boolean e2, boolean e3) {

```

Comment:
`boolean e1, boolean e2, boolean e3`
these are not needed anymore

Reply:
Done, deleted unnecessary code.